internal/stats/uptime.go

```diff
@@ -13,10 +13,26 @@ import (
 )
 
 func SubmitNewNodeUptimeReport(hactarClient *hactar.Client, lotusClient *lotus.Client, currentSession session.UserSession) bool {
-	_, err := lotusClient.Miner.GetMinerAddress()
-
+	isWorking := true
+	// check for miner address
+	miner, err := lotusClient.Miner.GetMinerAddress()
+	if err != nil {
+		isWorking = false
+	} else {
+		// check for last tipset
+		tipset, err := lotusClient.Blocks.GetLastTipset()
+		if err != nil {
+			isWorking = false
+		}
+		// get actor
+		if err == nil {
+			actor, _ := lotusClient.Miner.GetActor(miner, tipset.Cids[0])
```

Comment:
is `null` not supported?

Reply:
I am not sure what do you mean? are you referencing omitted error variable?